src/gwc/src/test/java/org/geoserver/gwc/GWCIntegrationTest.java

```diff
@@ -2086,4 +2087,30 @@ private void createLayerGroup(String layerGroupName, QName... layersNames) throw
         catalogBuilder.calculateLayerGroupBounds(layerGroup);
         getCatalog().add(layerGroup);
     }
+
+    /**
+     * Test verifying that GWC integration will work when layer belongs to default workspace and
+     * layer name is passed without workspace being part of layer name or OWS request itself
+     */
+    @Test
+    public void testDirectDefaultWorkspaceWMSIntegration() throws Exception {
```

Comment:
I would also add a test for when a virtual service is used and the provided layer name is not qualified, maybe this one already exists. I would also add the same tests for layer groups.